be/src/agent/task_worker_pool.cpp

```diff
@@ -402,6 +404,17 @@ void TaskWorkerPool::_drop_tablet_worker_thread_callback() {
             _tasks.pop_front();
         }
 
+        scoped_refptr<Trace> trace(new Trace);
+        MonotonicStopWatch watch;
+        watch.start();
+        SCOPED_CLEANUP({
+            if (watch.elapsed_time() / 1e9 > config::agent_task_trace_threshold_sec) {
+                LOG(WARNING) << "Trace:" << std::endl << trace->DumpToString(Trace::INCLUDE_ALL);
+            }
+        });
```

Comment:
Better to define a macro for these code, it's duplicate with code in ceate tablet, and if you add more traces for other tasks, there will be manu duplicate code.

Reply:
update